### PR TITLE
Updated to use [SecureContext] extended attribute.

### LINKED ISSUE #22: Spec should use [SecureContext] once that is defined in Web IDL
This issue comes from WICG/paymentrequest#58. Adding [SecureContext] to Web IDL is being tracked in heycam/WEBIDL#65.

### PR DESCRIPTION
**Pull request by adrianba**
Fixes #22.

### CODE REVIEW

**Comment by adrianhopebailie** [comment]
Editorially LGTM

Are we in consensus that secure context is a requirement? I believe so but can't recall if that was resolved.

**Comment by zkoch** [comment]
I believe there is consensus, yes. This conversation has come up numerous times, but we can always add to Thursday's agenda to close it out officially.

**Comment by adrianhopebailie** [comment]
@zkoch I think you're correct.
I keep confusing this with top-level context which I think is still not clear

**Comment by zkoch** [comment]
Yeah, I think top-level is #2.

**Comment by msporny** [comment]
+1 to merge this PR.

**Comment by adrianhopebailie** [comment]
+1 to merge

**Comment by adrianhopebailie** [comment]
@adrianba I am not putting this on the agenda for discussion on 5 May, I believe there is consensus to merge.

File: specs/paymentrequest.html [hunk 1]
```diff
@@ -216,7 +216,8 @@
         <dd>When this specification says to <dfn>throw</dfn> an error, the <a>user agent</a> must throw an
         error as described in [[!WEBIDL]]. When this occurs in a sub-algorithm, this results in
         termination of execution of the sub-algorithm and all ancestor algorithms until one is
-        reached that explicitly describes procedures for catching exceptions.</dd>
+        reached that explicitly describes procedures for catching exceptions.
+        <p>The term <dfn>extended attribute</dfn> is defined by [[!WEBIDL]].</dd>
```
Comment (davidlehn) [inline specs/paymentrequest.html:220]: Missing the closing `</p>`.

**Comment by adrianba** [comment]
Merged as 88ad7a0765bb36b0eca134868c98eabfd49c9e1c.

**Pull request closed without merging**